docs/setup.md

```diff
@@ -34,29 +39,42 @@ of Maven prior to build is *not* required.
+Prior to performing analysis in the ClintLR GUI, the locations of certain resources (the Mondo ontology, output directory, 
+LIRICAL data directory, and Exomiser variant files)
 need to be specified. This can be done from the Resources interface, accessed from the *File -> Show Resources* menu.
-Resources set in this interface are preserved between sessions.
+Resources set in this interface are preserved between sessions and saved in a .clintlr configuration folder that is 
+created automatically in the home directory.
 
-![Screenshot of the ClintLR resources interface](_static/ClintLRresourcesFigure.png "Screenshot of the ClintLR resources interface Ontology + Output tab (top) and LIRICAL tab (bottom)")
+<figure markdown>
+  ![Screenshot of the ClintLR resources interface](_static/ClintLRresourcesFigure.png)
+  <figcaption>Screenshot of the ClintLR resources interface Ontology + Output tab (top) and LIRICAL tab (bottom)</figcaption>
+</figure>
 
-The Mondo ontology and output directory paths are found in the *Ontology + Output* tab, and LIRICAL resources are found in the *LIRICAL* tab.
-If you have a local Mondo ontology file to use, you can set its path using the **Set Path** button. Alternatively, clicking
+### Ontology + Output
+The Mondo ontology and output directory paths are found in the *Ontology + Output* tab.
+The Mondo ontology is required to use ClintLR. Clicking
 the **Download** button will download and save the latest version of the Mondo ontology to your machine.
 
+**Note:**  There are currently no progress bars for the download buttons, however they are working in the background, and need
+only be clicked once.
+
 The output directory refers to a local directory to which the analysis results files will be saved.
 This can be set using the **Set Path** button next to the Output directory field.
 
+### LIRICAL 
+LIRICAL resources are found in the *LIRICAL* tab.
 The LIRICAL data directory refers to the directory that contains
-[additional files](https://thejacksonlaboratory.github.io/LIRICAL/stable/setup.html#rstsetup) that LIRICAL requires to run.
-Similar to the Mondo ontology, a local directory can be specified or the files can be downloaded.
+[additional files](https://thejacksonlaboratory.github.io/LIRICAL/stable/setup.html#lirical-data-files) that LIRICAL requires to run.
+Similar to the Mondo ontology, clicking
+the **Download** button will download and save the LIRICAL files to your machine.
 
-The Exomiser variant files refer to data files from Exomiser that LIRICAL uses to the annotate the VCF file and support variant interpretation.
+The Exomiser variant files refer to data files from Exomiser that LIRICAL uses to the annotate the VCF file and 
+support variant interpretation.
 The Exomiser data files can be obtained from the [Exomiser download site](https://exomiser.monarchinitiative.org/exomiser/download).
 Once the Exomiser files are downloaded, their local paths can be set using the respective **Set Path** buttons.
```

Comment:
Please point out that ClintLR only needs the variant database files. This will help the users to set path to the expected file, instead of, say, the other database file shipped with Exomiser resources.

An example doesn't hurt. For instance:

```
In the last tested Exomiser release `2309`, the variant database names are:

- **hg38**: 2309_hg38_variants.mv.db`
- **hg19**: 2309_hg19_variants.mv.db`
```

Reply:
Ok, thanks.